tests/gui/conversation/export/test_device.py

```diff
@@ -2,87 +2,59 @@
-            sync_thread=sync_thread,
-            main_queue_thread=main_queue_thread,
-            file_download_queue_thread=file_download_queue_thread,
-        )
-        device = Device(controller, export_service_thread=export_thread)
-        print_preflight_check_requested_emissions = QSignalSpy(
-            device.print_preflight_check_requested
-        )
-
-        device.run_printer_preflight_checks()
-
-        assert len(print_preflight_check_requested_emissions) == 1
+def no_session():
+    pass
 
 
-def test_Device_run_printer_preflight_checks_not_qubes(homedir, mocker, session, source):
+def test_Device_run_printer_preflight_checks(homedir, mocker, source):
     gui = mocker.MagicMock(spec=Window)
-    with threads(4) as [export_thread, sync_thread, main_queue_thread, file_download_queue_thread]:
+    with threads(3) as [sync_thread, main_queue_thread, file_download_queue_thread]:
         controller = Controller(
             "http://localhost",
             gui,
-            session,
+            no_session,
             homedir,
             None,
-            export_thread=export_thread,
             sync_thread=sync_thread,
             main_queue_thread=main_queue_thread,
             file_download_queue_thread=file_download_queue_thread,
         )
-        device = Device(controller, export_service_thread=export_thread)
-        controller.qubes = False
+        export_service = export.Service()
+        device = Device(controller, export_service=export_service)
         print_preflight_check_requested_emissions = QSignalSpy(
             device.print_preflight_check_requested
         )
-        print_preflight_check_succeeded_emissions = QSignalSpy(
-            device.print_preflight_check_succeeded
-        )
 
         device.run_printer_preflight_checks()
 
-        assert len(print_preflight_check_requested_emissions) == 0
-        assert len(print_preflight_check_succeeded_emissions) == 1
+        assert len(print_preflight_check_requested_emissions) == 1
 
 
-def test_Device_run_print_file(mocker, session, homedir):
+def test_Device_run_print_file(mocker, homedir):
     gui = mocker.MagicMock(spec=Window)
-    with threads(4) as [export_thread, sync_thread, main_queue_thread, file_download_queue_thread]:
+    with threads(3) as [sync_thread, main_queue_thread, file_download_queue_thread]:
         controller = Controller(
             "http://localhost",
             gui,
-            session,
+            no_session,
             homedir,
             None,
-            export_thread=export_thread,
             sync_thread=sync_thread,
             main_queue_thread=main_queue_thread,
             file_download_queue_thread=file_download_queue_thread,
         )
-        device = Device(controller, export_service_thread=export_thread)
-        device._export_service = mocker.MagicMock(spec=Export)
+        export_service = export.Service()
+        device = Device(controller, export_service=export_service)
```

Comment:
Note how we can use a real `export.Service` instance in this unit tests too. :slightly_smiling_face: